src/core/MeshRefractionMaterial.tsx

```diff
@@ -70,9 +70,7 @@ export function MeshRefractionMaterial({
     // Update the BVH
     if (geometry) {
       ;(material.current as any).bvh = new MeshBVHUniformStruct()
-      ;(material.current as any).bvh.updateFrom(
-        new MeshBVH(geometry.clone().toNonIndexed(), { lazyGeneration: false, strategy: SAH })
-      )
+      ;(material.current as any).bvh.updateFrom(new MeshBVH(geometry.clone().toNonIndexed(), { strategy: SAH }))
```

Comment:
`lazyGeneration` was removed in https://github.com/gkjohnson/three-mesh-bvh/pull/196